### ISSUE TITLE
x86 version doesn't work in Windows XP

### ISSUE THREAD

**Issue by shadowlmd** [colorer/FarColorer#78]
Hello,

x86 is not working in Windows XP (again).

![image](https://github.com/colorer/FarColorer/assets/969156/90111c2a-896c-4e35-a7ce-7ce231629f0e)

**Comment by yegor-mialyk** [comment]
Please take a look at Readme document:
```
FarColorer has two main verions:
 * 1.3.x - version with Windows XP support. Latest version included in Far Manager 1.3.26 in Far Manager 3.0.5796 (2021.05.09). This version has partial support. See in branch v1.3.x.
 * 1.4.x and higher - current fully supported version, worked only on actual platforms. It included in Far Manager since 3.0.5797 (2021.05.10).
```
https://github.com/colorer/FarColorer/blob/2b688d7fcf1444bd397b190deb808721aa75cb15/README.md?plain=1#L15C33-L15C33

**Comment by shadowlmd** [comment]
Got it, thank you.
